sa3883.md

```diff
@@ -6,12 +6,11 @@ UNI: sa3883
 
 Github: [link](https://github.com/joon0110)
 
+GitHub README: [link](https://github.com/joon0110/joon0110/blob/main/README.md)
+
 
 ## Project Proposal
-Add a feature to [Swim Cloud](https://www.swimcloud.com) that would help the user.
+Add a feature to [Swim Cloud](https://www.swimcloud.com) that would help the user but still in process.
```

Comment:
add link to `ranking.py.md` here

Reply:
Thank you for finding this out. I forgot to add the link here.